src/test/java/com/exasol/errorcodecrawlermavenplugin/ErrorCodeCrawlerMojoIT.java

```diff
@@ -216,6 +253,59 @@ static class ITVerifier {
                 throw new IllegalStateException("Failed to create test projects src folder.");
             }
             withConfiguration("testProject/" + CONFIG_NAME);
+            if (withSubProject) {
+                withSubProject();
+            }
+        }
+
+        ITVerifier withSubProject() throws IOException {
+            this.subProjectDir = this.projectDir.resolve("sub-project");
+            if (!(this.subProjectDir.toFile().mkdirs())) {
+                throw new IllegalStateException("Failed to create sub-project folder");
+            }
+            this.subProjectMainSrcJava = this.subProjectDir.resolve(Path.of("src", "main", "java"));
```

Comment:
```suggestion
            this.subProjectMainSrcJava = this.subProjectDir.resolve("src/main/java");
```
You can simplify this code, same below.

Reply:
In general, different operating systems might use different file and path separators. You can see some examples here: https://www.baeldung.com/java-file-vs-file-path-separator
So, in general, it is safer to pass a list of path directories and let Path class take care about adding File or Path separator, correspondent to current operating system. Path class will use File.pathSeparator instead of hardcoding "/".

Reply:
Actually, if it is the code in the test, then I think it is ok to hardcode file separator as "/".
In case it will ever cause the issue with unit tests on some operating systems, then maybe we might reconsider this approach. But for now, I think it is ok to hardcode "/" in the tests.